controlpanel/frontend/views/app.py

```diff
@@ -250,8 +275,12 @@ class RemoveCustomer(UpdateApp):
     def perform_update(self, **kwargs):
         app = self.get_object()
         user_ids = self.request.POST.getlist('customer')
-        app.delete_customers(user_ids)
-        messages.success(self.request, f"Successfully removed customer{pluralize(user_ids)}")
+        try:
+            app.delete_customers(user_ids)
+        except App.DeleteCustomerError:
+            messages.error(self.request, f"Failed removing customer{pluralize(user_ids)}")
```

Comment:
we probably want the nerdy details in Sentry or at least in the logs ☺️ (as long as we're confident the exception doesn't contain sensitive information)

Reply:
sentry strips sensitive information